src/components/containers/menu/MenuPanel.js

```diff
@@ -26,26 +25,26 @@ const StyledChildrenContainer = styled.div`
 `;
 
 const MenuPanel = (props, context) => {
-  const { children, isOpen, closeFunction } = props;
-
+  const { children, isOpen, closeFunction, theme } = props;
   return (
-    <ThemeProvider theme={defaultTheme}>
-      <StyledPanel isOpen={isOpen}>
-        <Spacer>
-          <StyledDismissButton onClick={closeFunction} />
-        </Spacer>
+    <StyledPanel isOpen={isOpen} className="menu-panel">
+      <Spacer>
+        <StyledDismissButton onClick={closeFunction} />
+      </Spacer>
+      <ThemeProvider theme={theme}>
```

Comment:
We don't actually need the ThemeProvider here if MenuPanel will always be wrapped by a Menu component.  The Menu ThemeProvider will pass the theme down to all descendant subcomponents.